### PR TITLE
added permissive param to provided searches

### PR DESCRIPTION
**Pull request by ryanhertz**
@joshcom The `:RequestMode => 'permissive'` parameter has been added to provided searches.

### CODE REVIEW

File: spec/unit/spark_api/models/saved_search_spec.rb [hunk 1]
```diff
@@ -139,18 +160,14 @@
   describe "can_have_newsfeed?" do
```
Comment (ryanhertz) [inline spec/unit/spark_api/models/saved_search_spec.rb]: The changes below are unrelated, but I had the file open so I figured I might as well fix these things.

File: spec/unit/spark_api/models/saved_search_spec.rb [hunk 2]
```diff
@@ -99,6 +99,27 @@
...
+      it "should return the searches listings" do
+        stub_api_get("/#{subject.class.element_name}/#{id}", 'saved_searches/get.json')
+        stub_api_get("/listings", 'listings/multiple.json', 
+          {:_filter => "City Eq 'Moorhead' And MlsStatus Eq 'Active' And PropertyType Eq 'A'"})
+        listings = subject.class.find(id).listings
+        listings.should be_an(Array)
+        listings[0].should be_a(Listing)
+      end
+      
+      it "should include the permissive parameter for provided searches" do
```
Comment (joshcom) [inline spec/unit/spark_api/models/saved_search_spec.rb]: I'd recommend a test that ensures `:RequestMode => 'permissive'` is not included for non-provided searches.

Comment (ryanhertz) [inline spec/unit/spark_api/models/saved_search_spec.rb]: @joshcom Done.

**Comment by joshcom** [comment]
:+1:

**Pull request merged**